Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -52,9 +52,9 @@ void GeneralConfigPane::InitializeGUI()
   for (const CPUCore& cpu_core : m_cpu_cores)
     m_cpu_engine_array_string.Add(cpu_core.name);
 
-  m_dual_core_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Dual Core (speedup)"));
+  m_dual_core_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Dual Core (Speedup)"));
   m_cheats_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Cheats"));
-  m_force_ntscj_checkbox = new wxCheckBox(this, wxID_ANY, _("Force Console as NTSC-J"));
+  m_force_ntscj_checkbox = new wxCheckBox(this, wxID_ANY, _("Force Console As NTSC-J"));
```

Comment:
It's debatable but I believe as usually gets capitalized in a title. See "Subordinating conjunctions": http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html for example